clusterloader2/pkg/measurement/common/kube_proxy_iptables_restore_failures.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Do we need to create dedicated measurement for this metric? If you want to simply gather value, show in perf dash, fail if exceeds threshold, then probably Generic Prometheus measurement is a way to go.

Reply:
https://github.com/kubernetes/perf-tests/blob/34b4a7a772ea2293a82ae13a1dc29fe19bd439cd/clusterloader2/testing/load/modules/measurements.yaml#L162

Reply:
I was just following the docs, no special need AFAIK ... I really think you'll know better ... the goal is to fail if that metrics is > 0 ...

Reply:
done